content/src/content/jcr_root/apps/core/wcm/sandbox/commons/ui/clientlibs/panelcontainer/js/PanelContainer.js

```diff
@@ -164,6 +176,15 @@
                 }
             });
         }
+
+        /**
+         * Triggered when the Panel Container has navigated.
+         *
+         * @event CQ.CoreComponents.PanelContainer#cmp-panelcontainer-navigated
+         * @param {Object} event Event object.
+         * @param {String} event.id The Panel Container ID (path)
+         * @param {Number} event.index Index of the navigated to Panel Container item
```

Comment:
The description is a bit unclear. Maybe something like: 'Index of the Panel Container item that is navigated to'?